jujugui/static/gui/src/app/store/env/bakery.js

```diff
@@ -344,15 +352,18 @@ YUI.add('juju-env-bakery', function(Y) {
     },
 
     /**
-      Interact to be able to sign-in to get authenticated.
-
-      @method _interact
-      @param {Function} The callback function to be sent in case of
-                        successful authentication
-      @param {Function} The callback function failure in case of
-                        wrong authentication
-      @return {Object} The asynchronous request instance.
-    */
+     Interact to be able to sign-in to get authenticated.
+
+     @method _interact
+     @param {Function} The callback function to be sent in case of
+            successful authentication.
+     @param {Function} The callback function failure in case of
+            wrong authentication.
+     @param {Function} The callback function failure in case of
+            wrong authentication.
+     @param {Object} e an xhr response object.
```

Comment:
Also, it would probably be worth being consistent in whether we use {Object} or {object}. Let's choose one and stick to it.